### PR TITLE
Make Settings page look nicer

### PR DESCRIPTION
**Pull request by Sprooty**
My first child

### CODE REVIEW

**Comment by OSPFNeighbour** [comment]
Screenshot

**Comment by Sprooty** [comment]
@OSPFNeighbour  capture

**Comment by sdunster** [comment]
Need to see what it looks like on iOS. Someone build and get a screenshot else I'll do it later. I will try to be more android aware in my changes too :)

File: client/src/screens/settings.screen.js [hunk 1]
```diff
@@ -148,28 +160,22 @@ class Settings extends Component {
             <TouchableOpacity style={styles.imageContainer}>
               <Image
                 style={styles.userImage}
-                source={{ uri: 'https://facebook.github.io/react/img/logo_og.png' }}
+                source={{ uri: 'https://www.lehigh.edu/google/google_contacts.png' }}
```
Comment (sdunster) [inline client/src/screens/settings.screen.js]: I'm going to use an icon for this instead - loading icon from random website probably not a great idea.

**Comment by sdunster** [comment]
screen shot 2017-11-16 at 21 58 07

**Comment by sdunster** [comment]
my tweaks are on #32

**Pull request closed without merging**